uniffi_bindgen/src/bindings/swift/templates/Protocol.swift

```diff
@@ -1,5 +1,5 @@
 {%- call swift::docstring_value(protocol_docstring, 0) %}
-public protocol {{ protocol_name }}: AnyObject {
+public protocol {{ protocol_name }}: AnyObject, Sendable {
```

Comment:
The failure is probably caused by this `Sendable` requirement. I don't think it needs to be there. The `uniffi::Object` classes should be `Sendable`. It does not really matter if the protocol behind them should be `Sendable` or not.